pom.xml

```
@@ -164,6 +164,26 @@
           <artifactId>maven-clean-plugin</artifactId>
           <version>${maven-clean-plugin.version}</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <configuration>
+            <additionalOptions>-html5 --allow-script-in-comments</additionalOptions>
+            <javadocDirectory>${maven.multiModuleProjectDirectory}/javadoc-theme</javadocDirectory>
```

I'm not sure we should rely on this property, see [MNG-6589](https://issues.apache.org/jira/browse/MNG-6589).

Maven 4 will bring better options, though ([MNG-7038](https://issues.apache.org/jira/browse/MNG-7038)).

---

@scordio wasn't aware of that. Thanks for the links!

---

You could use the `rootDirectory` property that I introduced for a similar challenge with Spotless:

```suggestion
            <javadocDirectory>${rootDirectory}/javadoc-theme</javadocDirectory>
```

Would that help?

---

Done, thanks!